lib/puppet/defaults.rb

```diff
@@ -1452,7 +1452,7 @@ def self.initialize_default_settings!(settings)
         To turn off reports entirely, set this to `none`",
     },
     :exclude_unchanged_resources => {
-      :default => false,
+      :default => true,
```

Comment:
Hmm, curious why no tests failed. Maybe we need to add a test to assert that the report contains unchanged resources when the setting is set to `false`, analogous to the `true` case? 

https://github.com/puppetlabs/puppet/blob/178ff2e2e0508edd90b577b0d79cbe55f172da1c/spec/unit/transaction/report_spec.rb#L140-L174

Reply:
I've added a new test as you described, let me know if this is what you had in mind